.pre-commit-config.yaml

```diff
@@ -9,7 +9,7 @@ repos:
     rev: 22.6.0
     hooks:
     -   id: black
--   repo: https://github.com/humitos/mirrors-autoflake
+-   repo: https://github.com/PyCQA/autoflake
```

Comment:
The old link no longer works (404 error). Maybe intermittent but moving to the official project for now.